### PR TITLE
Arb mainnet

### PR DESCRIPTION
**Pull request by dbeal-eth**

### CODE REVIEW

**Review by noisekit — APPROVED** [review]

File: tomls/collaterals/weth.toml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-include = ["../oracles/eth.toml"]
```
Comment (noisekit) [inline tomls/collaterals/weth.toml:1]: Sepolia and polygon failing because of this

**Review by noisekit — CHANGES_REQUESTED** [review]
Cannon change global weth config

**Comment by dbeal-eth** [comment]
I made the change to the global config because we probably should be migrating all the feeds over to pyth anyway

**Comment by noisekit** [comment]
> I made the change to the global config because we probably should be migrating all the feeds over to pyth anyway

Then we need to deprecate and delete old configs for sepolia and polygon or provide a way to migrate. As part of this pr.

File: tomls/collaterals/weth.toml (hunk 1 above)
Comment (noisekit) [reply tomls/collaterals/weth.toml:1]: fixed by removing nested includes

File: tomls/omnibus-arbitrum-mainnet-arbthetix/perps/feeCollector.toml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+[setting.pythSnxFeedId]
+defaultValue = "0x39d020f60982ed892abbcd4a06a276a9f9b7bfbce003204c110b6e488f502da3"
+
+[provision.buyback_snx]
+source = "<%= settings.buyback_snx_package %>"
+targetPreset = "<%= settings.target_preset %>"
+options.premium = "<%= settings.buyback_premium %>"
+options.snx_fee_share = "<%= settings.buyback_fee_share %>"
+options.oracle_manager_address = "<%= imports.system.imports.oracle_manager.contracts.Proxy.address %>"
+options.snx_token = "<%= imports.snx_mock_collateral.contracts.MintableToken.address %>"
```
Comment (noisekit) [inline tomls/omnibus-arbitrum-mainnet-arbthetix/perps/feeCollector.toml:21]: we have plenty of references to `mock` in these configs.

**Review by noisekit — APPROVED** [review]
Fixed outstanding issues and make this PR non-breaking. Plus fixed few more copy-paste issues.

As it is deployed now and there is no cannon diff anymore - we can merge and all other changes will need new PRs.

**Pull request merged**